dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/JspOptions.java

```diff
@@ -117,15 +115,15 @@ public class JspOptions {
     String overriddenJspOptions = new String();
     
     public JspOptions() {
-        setJdkSourceLevel(16);
+        setJavaSourceLevel(8); // Changed in PR 25494
     }   
     
     public JspOptions(Properties jspParams) {
-        setJdkSourceLevel(16);
+        setJavaSourceLevel(8); // Changed in PR 25494
```

Comment:
I changed this since newer releases (i.e. 21) require Java 8 now.  Java 17 requires a min of 7.     I don't see a reason to keep 16 as the default anymore.